packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+---
+title: Peering integration guide
```

Comment:
The other requirements pages are simply titled "Assets" or "Wallet addresses". Do you think we should follow the same format here, or maybe the other page titles should be updated?

Reply:
Changed to "Peers"